internal/drivers/go-duckdb.go

```diff
@@ -0,0 +1,9 @@
+package drivers
```

Comment:
filename nit: duckdb.go has nothing to do with go =)

Reply:
it's the name of the package so i was following the naming convention used with pgx 🤣 will fix

Reply:
*smile* at one time there were two (!!) postgresql drivers in genieql. one was discontinued so I removed it. =)